### PR TITLE
github: use yml template

### PR DESCRIPTION
**Pull request by docsir**
### What problem does this PR solve? 
None

### What is changed and how it works?
Use yml template to get better experience

### Check List 

Tests 
 - No code

Code changes
None

Side effects
None

Related changes
None

Release note
`None`

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    attributes:
+      label: What did you expect to see?
+    validations:
+      required: false
+  - type: textarea
+    id: see
+    attributes:
+      label: What did you see instead?
+    validations:
+      required: true
```
Comment (glorv) [inline .github/ISSUE_TEMPLATE/bug-report.yml:27]: I thing we should only make `What did you do?` section as required and leave all the following optional.

Comment (docsir) [reply .github/ISSUE_TEMPLATE/bug-report.yml:27]: Thanks for your comment. The type of this issue is "bug", so I think the other sections also should be required. For `What did you expect to see?` section, I have no idea.

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+             ```
+             Upstream MySQL/MariaDB server version:
+             ```console
+             (paste upstream MySQL/MariaDB server version here)
+             ```
+             Downstream TiDB cluster version (execute `SELECT tidb_version();` in a MySQL client):
+             ```console
+             (paste TiDB cluster version here)
+             ```
+             How did you deploy DM: DM-Ansible or manually?
```
Comment (Ehco1996) [inline .github/ISSUE_TEMPLATE/bug-report.yml:45]: We should taper our support for ansbile
Perhaps it would be better to replace ansbile with `tiup` because there is no ansible-deploy section in official doc

File: go.mod [hunk 3]
```diff
@@ -27,7 +27,7 @@ require (
 	github.com/pingcap/log v0.0.0-20210906054005-afc726e70354
 	github.com/pingcap/parser v0.0.0-20210907051057-948434fa20e4
 	github.com/pingcap/tidb v1.1.0-beta.0.20210914112841-6ebfe8aa4257
-	github.com/pingcap/tidb-tools v5.2.0-alpha.0.20210727084616-915b22e4d42c+incompatible
+	github.com/pingcap/tidb-tools v5.2.1+incompatible
```
Comment (Ehco1996) [inline go.mod:30]: why update dependency in here? this may cause conflicts with other PR 🤣
please remove this part, if you want to upgrade dep, you can do it in another PR

File: .github/ISSUE_TEMPLATE/bug-report.yml (hunk 2 above)
Comment (docsir) [reply .github/ISSUE_TEMPLATE/bug-report.yml:45]: I looked up the official doc for deploy DM, and it's recommended to deploy DM with tiup.
But for `manually`, I'm not sure if this needs to be kept.

File: go.mod (hunk 3 above)
Comment (docsir) [reply go.mod:30]: It's automaticlly changed by vscode, and I forgot to check this.
Thanks for your advice, I will remove this part in next commit.

File: .github/ISSUE_TEMPLATE/bug-report.yml (hunk 2 above)
Comment (lichunzhu) [reply .github/ISSUE_TEMPLATE/bug-report.yml:45]: I agree switch to tiup

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+  - type: textarea
+    id: current
+    attributes:
+      label: current status of DM cluster (execute `query-status` in dmctl)
+      value: |-
+             ```console
+             (paste current status of DM cluster here)
+             ```
+    validations:
+      required: true
```
Comment (lichunzhu) [inline .github/ISSUE_TEMPLATE/bug-report.yml:65]: I think it's not required

**Review by lichunzhu — APPROVED** [review]
LGTM

**Review by okJiang — APPROVED** [review]
